src/auth/ui/vue/show.ts

```diff
@@ -843,3 +843,11 @@ export async function emitWebviewClosed(authWebview: ClassToInterfaceType<AuthWe
         return result
     }
 }
+
+/**
+ * Forces focus to Amazon Q panel - USE THIS SPARINGLY (don't betray customer trust by hijacking the IDE)
+ * Used on first load, and any time we want to directly populate chat.
+ */
+export async function focusAmazonQPanel(): Promise<void> {
```

Comment:
Did this need to be moved here, was it to fix a circular dependency issue?

Reply:
Yes, something had to be moved from one of the 2 files, of which both things were reference across multiple other files. This made sense IMO but was arbitrary.